### PR TITLE
more code cleanup

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/modifier/set/AddModifierFactory.java [hunk 1]
```diff
@@ -121,16 +122,16 @@ public String getInstructions()
...
 		/**
 		 * The objects to be added to the active set when this AddModifier is
 		 * processed
 		 */
 		private Indirect<T[]> toAdd;
 
-		public AddIndirectArrayModifier(FormatManager<T[]> formatManager,
-			int userPriority, Indirect<T[]> toAdd)
+		private AddIndirectArrayModifier(FormatManager<T[]> formatManager,
+		                                 int userPriority, Indirect<T[]> toAdd)
```
Comment (thpr) [inline code/src/java/plugin/modifier/set/AddModifierFactory.java]: I'm concerned this indent is a formatting mismatch that may trigger churn

Comment (grimreaper) [inline code/src/java/plugin/modifier/set/AddModifierFactory.java]: likely is. What should I set my IDE to?

Comment (thpr) [inline code/src/java/plugin/modifier/set/AddModifierFactory.java]: I have to admit I don't know the exact knob, which is why we have the standards file for formatting.

**Comment by grimreaper** [comment]
killed in favor of other cleanups

**Pull request closed without merging**